src/index.ts

```diff
@@ -1,9 +1,8 @@
-import type { Plugin } from 'vite'
 import { Options, Transformer } from './types'
 import { Context } from './context'
 import { parseId } from './utils'
-import { Vue3Transformer } from './transforms/vue3'
-import { Vue2Transformer } from './transforms/vue2'
+import { getTransformer } from './transforms/transformers'
```

Comment:
You can change filename to index.ts that only need to import ./transforms

Reply:
done